src/main/groovy/wooga/gradle/appcenter/tasks/AppCenterTaskSpec.groovy

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018-2021 Wooga GmbH
```

Comment:
This class was generated today so the year must be 2022

Reply:
Got it.